Solutions/Digital Shadows/Playbooks/DigitalShadowsPlaybook-UpdateIncidentStatus.json

```
@@ -0,0 +1,408 @@
+{
+    "$schema":  "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion":  "1.0.0.0",
+    "metadata":  {
+        "title":  "Digital Shadows Playbook to Update Incident Status",
+        "description":  "The playbook will update the status of Azure incidents which are created from the Alerts triggered by Digital Shadows SearchLight portal",
+        "prerequisites":  "The Digital Shadows Data Connector is deplyed on your environment",
```

Looking at https://github.com/Azure/Azure-Sentinel/blob/master/MasterPlaybooks/Remediation-IP/azuredeploy.json for comparison.

```suggestion
        "prerequisites":  [
            "The Digital Shadows Data Connector is deployed on your environment"
        ],
```